File: Android Layouts/SMD_assignment2/app/src/main/java/com/smd/smd_assignment2/MainActivity.java
package com.smd.smd_assignment2;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RelativeLayout;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        switch_constraint();
        switch_relative();
    }
      private void switch_constraint()
        {
            Button constraint;
            constraint=(Button)findViewById(R.id.constraint_button);
            constraint.setOnClickListener(new View.OnClickListener() {
                @Override
                public void onClick(View view) {
                startActivity(new Intent(MainActivity.this,Constraint_Layout.class));
                }
            });
        }

        private void switch_relative()
        {
            Button relative;
            relative=(Button)findViewById(R.id.relative_button);
            relative.setOnClickListener(new View.OnClickListener() {
                @Override
                public void onClick(View view) {
                    startActivity(new Intent(MainActivity.this, Relative_Layout.class));
                }
            });
        }
}

File: Android Layouts/assignment2_SMD/app/src/main/java/com/smd/assignment2_smd/MainActivity.kt
package com.smd.assignment2_smd

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.RadioButton
import android.widget.Toast
import com.smd.assignment2_smd.R.layout.signup_constraint
import kotlinx.android.synthetic.main.signup_relative.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.screen_linear)

        val email=findViewById<EditText>(R.id.Email)
        val password=findViewById<EditText>(R.id.Password)
        val male=findViewById<RadioButton>(R.id.radioButton)
        val female=findViewById<RadioButton>(R.id.radioButton2)

    val relative=findViewById<Button>(R.id.button3)
        relative.setOnClickListener {
            setContentView(R.layout.signup_relative)
            val register_relative=findViewById<Button>(R.id.Register_Relative)
            register_relative.setOnClickListener {
                Toast.makeText(this,email.text,Toast.LENGTH_LONG).show()
                Toast.makeText(this,password.text,Toast.LENGTH_LONG).show()
            }
        }
        val constraint=findViewById<Button>(R.id.button4)
        constraint.setOnClickListener {
            setContentView(R.layout.signup_constraint)
            val register_constraint=findViewById<Button>(R.id.Register_Constraint)
            register_constraint.setOnClickListener {
                Toast.makeText( this,email.text,Toast.LENGTH_LONG).show()
                Toast.makeText(this,password.text,Toast.LENGTH_LONG).show()
            }
        }
    }
}
